src/PAModel/MsAppTest.cs

```diff
@@ -276,18 +277,35 @@ public static void CompareChecksums(string pathToZip, TextWriter log, Dictionary
                                 bool same = newContents.SequenceEqual(originalContents);
 
                                 if (!same)
-                                { 
-                                    var jsonDictionary1 = FlattenJson(originalContents);
-                                    var jsonDictionary2 = FlattenJson(newContents);
-
-                                    // Add JSONMismatch error if JSON property was changed or removed
-                                    CheckPropertyChangedRemoved(jsonDictionary1, jsonDictionary2, errorContainer, "");
-
-                                    // Add JSONMismatch error if JSON property was added
-                                    CheckPropertyAdded(jsonDictionary1, jsonDictionary2, errorContainer, "");
+                                {
+
+                                    bool mismatch = false;
+
+                                    // Catch in case of originalContents/newContents not being JSON
+                                    try
+                                    {
+                                        JsonDocument.Parse(originalContents);
+                                        JsonDocument.Parse(newContents);
+                                    }
+                                    catch
+                                    {
+                                        mismatch = true;
+                                    }
+
+                                    if (!mismatch)
+                                    {
+                                        var jsonDictionary1 = FlattenJson(originalContents);
```

Comment:
We could ideally move all the contents in this if block within the same try block above, right?
If JsonDocument.Parse throws exception, your code would catch it anyways I believe.

Reply:
The reason I made the try catch separate, is that, we want to make sure the exception being caught has to do with the JSON being invalid. Adding the lines below to the try block will make it so that 'any' exception will be caught, and not continue. We still want those to be handled properly